Amend grant_number --> grant_numbers. (#79)
jisc-services/Public-Documentation: PublicationsRouter/api/v3/IncomingNotification.md

				"id": "<funder identifier>"
				}
			],

> 			"grant_numbers": "<funder's grant number>"

Author: ```suggestion
			"grant_numbers":  ["<funder's grant numbers>"]
```

| metadata.funding.name | Funder name | unicode |  |  |
| metadata.funding.identifier.type | Funder identifier type (e.g "ringold") - no vocabulary for this field in this version of the system | unicode |  |  |
| metadata.funding.identifier.id | Funder identifier (e.g. Ringold ID) | unicode |  |  |

> | metadata.funding.grant_numbers | Grant number for funding source behind this article | unicode |  |  |

Author: ```suggestion
| metadata.funding.grant_numbers | Grant numbers funding the work of this article | unicode |  |  |
```
